script/client

```diff
@@ -47,6 +47,14 @@ Set API base URL component, default: '/api/v1'
 
 Output JSON instead of perl structures.
 
+=item B<--yaml-output>
+
+Output YAML instead of perl structures.
```

Comment:
I'd take the opportunity to improve the spelling of the comments and capitalize Perl.

Reply:
done